niviz_rater/validation.py

```diff
@@ -1,47 +1,78 @@
+import os
 import yaml
 import json
-import os
 import yamale
 from yamale.validators import DefaultValidators, Validator
-from bids.config import get_option
 
 SCHEMAFILE = os.path.join(os.path.dirname(__file__), 'data/schema.yaml')
 
 
-def load_json(file):
+def _load_json(file):
     with open(file, 'r') as f:
         result = json.load(f)
     return result
 
 
-def get_entities():
-    configfiles = [
-        load_json(file)['entities']
-        for file in get_option('config_paths').values()
-    ]
+def _get_valid_entities(config_files):
+    valid_configs = [_load_json(file)['entities'] for file in config_files]
     enumstrs = []
-    for configfile in configfiles:
+    for configfile in valid_configs:
         enumstrs.extend([entity['name'] for entity in configfile])
     return enumstrs
 
 
 class Entities(Validator):
     '''
     Class to enable validation of BIDS entities
+    from pyBIDS JSON configuration files
     '''
 
+    __slots__ = "valid_configs"
+
```

Comment:
Entities should check that `valid_configs` exists upon being initialized.

Reply:
great suggestion, I agree that a check should be implemented here since Entities by itself is actually not usable! Will add a check on init